Gulpfile.js

```diff
@@ -1,15 +1,30 @@
 var gulp        = require('gulp');
 var babel       = require('gulp-babel');
 var mocha       = require('gulp-mocha');
+var sequence    = require('gulp-sequence');
 var del         = require('del');
 var path        = require('path');
 var nodeVersion = require('node-version');
 var spawn       = require('./utils/spawn');
 
 
-var packageParentDir  = path.join(__dirname, '../');
-var packageSearchPath = (process.env.NODE_PATH ? process.env.NODE_PATH + path.delimiter : '') + packageParentDir;
+function gulpTestTask (taskName/*, ...args*/) {
+    var internalTaskName = taskName + '-internal';
```

Comment:
I don't like the idea to define some gulp tasks here implicitly. In fact, now I don't see what the tasks I can run when I look at the Gulpfile.js